formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/Hocon.kt

```diff
@@ -141,7 +141,9 @@ public sealed class Hocon(
             when {
                 descriptor.kind.listLike -> ListConfigReader(conf.getList(currentTag))
                 descriptor.kind.objLike -> if (ind > -1) ConfigReader(conf.getConfig(currentTag)) else this
-                descriptor.kind == StructureKind.MAP -> MapConfigReader(conf.getObject(currentTag))
+                descriptor.kind == StructureKind.MAP ->
+                    //if current tag is null - map in the root of config
```

Comment:
```suggestion
                    // if current tag is null - map in the root of config
```

IMO, it's more readable. Most comments in codebase a written with this style (although capitalization differs, we probably need standard on that too)